packages/gatsby-image/src/index.js

```diff
@@ -48,18 +48,67 @@ const convertProps = props => {
   return convertedProps
 }
 
+/**
+ * Checks if fluid or fixed are art-direction arrays.
+ * @param props   {object}   The props to check for images.
+ * @param prop    {string}   Check for fluid or fixed.
+ * @return {boolean}
+ */
+const hasArtDirectionSupport = (props, prop) =>
+  props[prop] &&
+  Array.isArray(props[prop]) &&
+  props[prop].some(image => typeof image.media !== `undefined`)
```

Comment:
Just curious; is `.media` an "art direction"?

Reply:
As the `media` property has to be added by a dev to use art-direction support, I guess we might use them synonymously ; ).